pkg/controller/instance/plan_execution_engine.go

```diff
@@ -174,12 +182,17 @@ func executeStep(step v1alpha1.Step, state *v1alpha1.StepStatus, resources []run
 		}
 
 		if allHealthy {
-			state.State = v1alpha1.PhaseStateComplete
+			state.Status = v1alpha1.ExecutionComplete
 		}
 	}
 	return nil
 }
 
+func prettyPrint(i interface{}) string {
+	s, _ := json.MarshalIndent(i, "", "  ")
```

Comment:
this _could_ fail for custom marshalers, make sure if this errors to set some reasonable default at least ("{}"?) (so you can preserve your return signature)